.gitignore

```diff
@@ -94,3 +94,10 @@ parser/parser.out
 parser/Release
 parser/lemon
 parser/lemon.exe
+
+# Eclipse
+.cproject
+.project
+.settings/
+.buildpath
+.tracefile
```

Comment:
What about https://help.github.com/articles/ignoring-files/#create-a-global-gitignore 

See top of this file :)